.travis.yml

```diff
@@ -25,7 +25,7 @@ before_install:
     - wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
```

Comment:
Yes, sorry, I wanted to unblock all the other branches waiting on travis tests!  We could, I don't feel strongly either way.